### ISSUE TITLE
Should there be a 'date' point-in-time metric?

### ISSUE THREAD

**Issue by skswong8** [humanmade/aws-analytics#274]
Was playing around with the point-in-time data metrics which are pretty cool

I added a case where I wanted todays date and time which would influence the related XB

Field : operator : value
```
Year : is greater than or equal to : 2021
Month : is greater than or equal to : 8
Day : is greater than or equal to : 4
Hour : is greater than or equal to : 7
```

But the `day` indicates Thursday, so it would be valid every Thursday, Friday, Saturday and Sunday of every week in August

I think `day` is still useful in cases and `date` would be great for timestamping too. What do you think?

**Comment by roborourke** [comment]
You could set them to be exactly equal to the target value. We do store `atrributes.date` which is a UTC date stamp.

By date in this context do you day of the month?

**Comment by skswong8** [comment]
By date I mean the numbered day of the month (01-31) correct

So for a silly example I wanted a condition which said now until say 01/01/2022 I want to display a XB which displays content related to 2021. After which that content changes to a 2022 version

As far as I could see, there isn't a way to do this currently with the fields provided? And since you have month and year, I thought it might be useful to have the day of the month too

![image](https://user-images.githubusercontent.com/3299318/135380148-5544427b-3861-4535-b82f-178d6080f2ed.png)

What do you think?

**Comment by roborourke** [comment]
Yeah it makes sense, the “day” label should be updated to “day of the week” and “day” or “date” added for the day of the month.

we also have a system for labelling the values now so we can make the day of the week and month values more user friendly
